packages/webshot/test.R

```diff
@@ -0,0 +1,3 @@
+install.packages("webshot", repos = "https://cran.rstudio.com")
+library(webshot)
+system("phantomjs --version")
```

Comment:
Please actually test that the `webshot` package works, not just that `phantomjs` binary is present.

Reply:
Yes. The first version of "test.R" does include the actual test cases but it keeps popping up the following message and never ends:


```
Garbage collection 6 = 3+1+2 (level 0) ... 
14.8 Mbytes of cons cells used (79%)
5.8 Mbytes of vectors used (73%)
Garbage collection 7 = 3+2+2 (level 1) ... 
15.9 Mbytes of cons cells used (85%)
```